examples/UKCore-RequestGroup-Paracetamol-Example.xml

```diff
@@ -0,0 +1,40 @@
+<RequestGroup xmlns="http://hl7.org/fhir">
+	<id value="UKCore-RequestGroup-Paracetamol-Example"/>
+	<status value="completed"/>
+	<intent value="order"/>
+	<priority value="routine"/>
+	<code>
+		<coding>
+			<system value="https://dmd.nhs.uk"/>
```

Comment:
This needs to be the snomed ct namingsystem

Reply:
done